examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,48 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+#author: Istvan szalai 22/05/2015
+
+fun core(player, computer: Int): Int do
+	if player == computer then return 0
+	if player > computer then return 1
+	return 2
+
+end
+
+fun resultMessage(number: Int): String do
```

Comment:
We don't encourage the use of camelCase in nit, generally variables and functions are expressed using snake_case